packages/design-system-docs/src/pages/components/FormLabel/form-label.example.html

```diff
@@ -1,9 +1,22 @@
-<label class="ds-c-label" for="number-field-id">
-  <span>Phone number</span>
-  <span class="ds-c-field__hint">
-    Optional. We'll use this number as a backup if we need to contact you about your application.
-  </span>
-  <span class="ds-c-field__hint ds-u-color--error" role="alert">
-    Please enter a valid phone number.
-  </span>
-</label>
+<div class="example--wrapper">
+  <label class="ds-c-label" for="number-field-id">
+    <span>Phone number</span>
+    <span class="ds-c-field__hint">
+      Optional. We'll use this number as a backup if we need to contact you about your application.
+    </span>
+    <span class="ds-c-field__hint ds-u-color--error" role="alert">
+      Please enter a valid phone number.
+    </span>
+  </label>
+  <div class="example--wrapper example--inverse">
+    <label class="ds-c-label" for="inverse-field-id">
+      <span>Inverse Example</span>
```

Comment:
Make `e` in example lower case
```suggestion
      <span>Inverse example</span>
```